src/libutil/paramlist.cpp

```diff
@@ -383,6 +384,22 @@ ParamValue::clear_value() noexcept
 
 
 
+size_t
+ParamValue::datasize() const noexcept
+{
+    return m_nvalues * static_cast<int>(m_type.size());
+}
```

Comment:
Note that this doesn't actually match the description you gave in the declaration -- this isn't necessarily heap memory.

Reply:
Fixed :+1: